zerver/tests/test_subs.py

```diff
@@ -5585,15 +5771,15 @@ def test_subscriptions_query_count(self) -> None:
         ]
 
         # Test creating a public stream when realm does not have a notification stream.
-        with self.assert_database_query_count(38):
+        with self.assert_database_query_count(40):
```

Comment:
This creates an extra query when creating a new stream using the following code snippet:
```
(stream, created) = Stream.objects.get_or_create(
        realm=realm,
        name__iexact=stream_name,
        defaults=dict(
            name=stream_name,
            creator=acting_user,
            description=stream_description,
            invite_only=invite_only,
            is_web_public=is_web_public,
            stream_post_policy=stream_post_policy,
            history_public_to_subscribers=history_public_to_subscribers,
            is_in_zephyr_realm=realm.is_zephyr_mirror_realm,
            message_retention_days=message_retention_days,
            can_remove_subscribers_group=can_remove_subscribers_group,
        ),
    )
```

How can I modify it so that when creating a new stream, it returns the stream with `can_remove_subscribers_group.namedusergroup.name`?

Reply:
No it did not helped.